uint16 x1
uint16 y1
uint16 x2
uint16 y2
uint8 class_id
float32 confidence
